airbyte-config/models/src/main/resources/types/IntegrationLauncherConfig.yaml

```diff
@@ -0,0 +1,18 @@
+---
+"$schema": http://json-schema.org/draft-07/schema#
```

Comment:
This is not really a config, right? It is just a data class to interact with Temporal. If so then it shouldn't be in this directory.

Reply:
agreed. will add it to my list of refactors for temporal.